src/Services/SolrCoreService.php

```diff
@@ -17,6 +17,8 @@
 use Firesphere\SolrSearch\Traits\CoreServiceTrait;
 use GuzzleHttp\Client as GuzzleClient;
 use GuzzleHttp\HandlerStack;
+use Http\Discovery\HttpClientDiscovery;
```

Comment:
Hi @Firesphere, 
Can we add `php-http/discovery:^1.14` to `composer.json`?
I got this error below:
```
[Emergency] Uncaught Error: Class 'Http\Discovery\HttpClientDiscovery' not found
```

Reply:
Yes, Of course!